torchvision/models/detection/transform.py

```diff
@@ -10,6 +10,37 @@
 from .roi_heads import paste_masks_in_image
 
 
+@torch.jit.unused
```

Comment:
cc @fmassa 
I implemented this separate code path to avoid constant sale_factor. Issue is describe in: https://github.com/pytorch/pytorch/issues
Since 'scale_factor' was used to resize the image as well as target['masks'], I had to factor out this part of the code together.